logging_config.ini

```diff
@@ -0,0 +1,39 @@
+[loggers]
+keys=root,fileLogger,sysLogger
+
+[handlers]
+keys=fileHandler,sysHandler
+
+[formatters]
+keys=customFormatter
+
+[logger_root]
+level=INFO
+handlers=fileHandler,sysHandler
+
+[logger_fileLogger]
+level=INFO
+handlers=fileHandler
+qualname=shaptools,hanadb_exporter
+propagate=0
+
+[logger_sysLogger]
+level=WARNING
```

Comment:
@arbulu89 By default, I would forward Warning and Errors to the log file too. It is easier to debug when it is all in the same file. I would also set Warn and Error only as the default, as the INFO is really verbose.

Other point is that you expect this file to be configurable (as it is being installed on /etc), so I would recommend to include comment headers on each section in order to instruct the users on how to use it.

Reply:
@diegoakechi Yes, right now up to INFO (what means, CRITICAL, ERROR, WARNING, INFO) are written in the log file and only up to WARNING in syslog. It's true, that it is quite verbose, we can reduce the level by default.

Ok, i will include some comments to explain how the configuration file works. Anyway, this is a standard python logging configuration file, so documentation is easy to find

Reply:
@arbulu89 Yes, but still I think some comments is nice to have and maybe a link pointing to more information from the official documentation.

Reply:
Working on that